store/postgres/tests/relational.rs

```diff
@@ -150,6 +150,16 @@ fn insert_entity(conn: &PgConnection, layout: &Layout, entity_type: &str, entity
     layout.insert(&conn, &key, entity, 0).expect(&errmsg);
 }
 
+fn update_entity(conn: &PgConnection, layout: &Layout, entity_type: &str, entity: Entity) {
+    let key = EntityKey {
+        subgraph_id: THINGS_SUBGRAPH_ID.clone(),
+        entity_type: entity_type.to_owned(),
+        entity_id: entity.id().unwrap(),
+    };
+    let errmsg = format!("Failed to update entity {}[{}]", entity_type, key.entity_id);
+    layout.update(&conn, &key, entity, 0).expect(&errmsg);
```

Comment:
To be truly an update, this should use block number 1, as we insert at block number 0. That will make sure that we actually have both the old and new entity in the store.